src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/DateTimeValue_TimeZone_Paris.txt

```diff
@@ -0,0 +1,40 @@
+﻿// (UTC+01:00) Brussels, Copenhagen, Madrid, Paris
+#SETUP: TimeZoneInfo("Romance Standard Time")
+
+>> DateTimeValue("2022-08-23T12:00:00Z") 
+DateTime(2022,8,23,14,0,0,0) 
+
+>> DateTimeValue("2022-08-23T12:00:00") 
+DateTime(2022,8,23,12,0,0,0)
+
+>> DateTimeValue("jeudi 21 juillet 2022 19:34:03", "fr-FR")
+DateTime(2022,7,21,19,34,3,0)
+
+>> DateTimeValue("jeudi 21 juillet 2022 19:34:03", "unknown")
+#Error(Kind=BadLanguageCode)
+
+// The result of this expression depends on current TimeZoneInfo. 
+// In Seattle TimeZoneInfo, this would trip over daylight savings and be 4:00am. 
+// In Paris timezoneinfo - which does not do daylight savings at the same date, we get 3:00am.
+// https://www.timeanddate.com/time/change/usa/seattle
+// https://www.timeanddate.com/time/change/france/paris
+>> Text(DateAdd(DateTime(2022,3,13,1,0,0), 2, "hours"))
+"3/13/2022 3:00 AM"
+
+>> Text(DateAdd(DateTime(2022,3,27,1,0,0), 2, "hours"))
+"3/27/2022 4:00 AM"
+
+>> DateAdd((Date(2022, 3, 13) + Time(1,0,0)), 90, "Minutes") 
+DateTime(2022,3,13,2,30,0,0)
+
+>> DateDiff(DateTime(2022,3,27,1,0,0), DateTime(2022,3,27,4,0,0), "hours")
+2
+
+>> DateDiff(DateTime(2022,3,13,1,0,0), DateTime(2022,3,13,4,0,0), "hours")
+3
+
+>> TimeZoneOffset(DateTime(2022,3,27,1,0,0))*24
```

Comment:
>TimeZoneOffset(DateTime(2022,3,27,1,0,0)) [](http://example.com/codeflow?start=3&length=41)

It's great that you added this test, as this exposed a bug that we have in the TimeZoneOffset implementation. It should "returns the **number of minutes** between the user's local time and UTC (Coordinated Universal Time)" - from [https://docs.microsoft.com/power-platform/power-fx/reference/function-dateadd-datediff](https://docs.microsoft.com/power-platform/power-fx/reference/function-dateadd-datediff).